site_scons/components/__init__.py

```diff
@@ -210,7 +211,7 @@ def define_mercury(reqs):
                                []))
 
     reqs.define('mercury',
-                retriever=GitRepoRetriever('https://github.com/mercury-hpc/mercury.git', True),
+                retriever=GitRepoRetriever('https://github.com/bwilsoncn/mercury.git', True),
```

Comment:
This will need to be pointed back to https://github.com/mercury-hpc/mercury.git.